firebase-vertexai/src/main/kotlin/com/google/firebase/vertexai/type/FunctionDeclaration.kt

```diff
@@ -36,11 +36,11 @@ package com.google.firebase.vertexai.type
  * @param optionalParameters A list of parameters that can be omitted.
  * @see Schema
  */
-class FunctionDeclaration(
-  val name: String,
-  val description: String,
-  val parameters: Map<String, Schema>,
-  val optionalParameters: List<String> = emptyList(),
+public class FunctionDeclaration(
+  internal val name: String,
```

Comment:
Why are these `internal` when most other API types have public fields?

Reply:
Since `FunctionDeclaration` is created by the dev, I'm moving it to internal. Objects created by the sdk have the fields set to public. wdyt?

Reply:
If you can move the class to internal I think that makes sense.

Reply:
For clarity, the `FunctionDeclaration` class will remain public, what's going internal are the fields in it.